cmd/launcher/desktop.go

```diff
@@ -185,15 +186,20 @@ func runDesktop(_ *multislogger.MultiSlogger, args []string) error {
 	}, func(err error) {})
 
 	// run run group
-	go func() {
+	gowrapper.Go(context.TODO(), slogger, func() {
 		// have to run this in a goroutine because menu needs the main thread
 		if err := runGroup.Run(); err != nil {
 			slogger.Log(context.TODO(), slog.LevelError,
 				"running run group",
 				"err", err,
 			)
 		}
-	}()
+	}, func(r any) {
+		slogger.Log(context.TODO(), slog.LevelError,
+			"exiting after runGroup panic",
+			"err", r,
+		)
+	})
```

Comment:
I don't think you need to provide a second function to `gowrapper.Go` just to perform logging -- the gowrapper already handles logging on panic. This `onPanic` function is only needed if we need to perform an additional action when we notice a panic. (See e.g. https://github.com/kolide/launcher/blob/main/cmd/launcher/svc_windows.go#L205.)

Anywhere in this PR you have an `onPanic` function that just does logging, I would remove it and replace with `func(r any) {}`.

Reply:
Done

Reply:
I think just this one got left behind when you were updating 🙂